dataplicity/m2m/bencode.py

```diff
@@ -60,12 +61,33 @@ def encode(obj):
 
     def add_encode(obj):
         if isinstance(obj, bytes):
-            append(b"{}:{}".format(len(obj), obj))
+            if PY2:
```

Comment:
We could avoid the PY2 branch entirely with something like the following, which should work with both Py versions...

```
append(u"{}:".format(len(obj)).encode())
append(obj)
```